dataprofiler/profilers/profiler_options.py

```diff
@@ -355,7 +365,7 @@ def _validate_helper(self, variable_path='FloatOptions'):
         if not isinstance(self.precision, BooleanOption):
```

Comment:
I'm a bit confused, I just looked at another PR from @JGSweets where precision was was an integer denoting sig figs, is this something different?

Reply:
Yes, I believe this is different: self.precision is a BooleanOption that will indicate if precision is enabled or not enabled.